framework/source/class/qx/util/ResourceManager.js

```diff
@@ -291,8 +298,16 @@ qx.Class.define("qx.util.ResourceManager",
           }
           // If the resourceUri begins with a single slash, include the current
           // hostname
-          else if (resourceUri.match(/^\//) != null) {
-            statics.__urlPrefix[lib] = window.location.protocol + "//" + window.location.host;
+          else if (resourceUri.match(/^\//) != null)
```

Comment:
@cajus @derrell OK bak to the current PR. I've just tried to fix all those unnecessary formatings besides the two spaces in empty lines. 
Is the PR now acceptable from this point of view?